Automattic/jetpack
# Calypsoify: Refactor and add autoupdate functionality

Iterations on the current pass.  Reorganize some assets, tidy up some formatting and code patterns, convert the css to scss to make for easier maintenance going forward and abstract out the color codes to allow for semantic renaming in a future pass.

Oh, and add autoupdate management functionality to the plugins page.

![pageshot of plugins test local wordpress thingy wordpress 2018-11-09-0843 24](https://user-images.githubusercontent.com/941023/48265919-d8e54280-e3fb-11e8-96db-3b789a7a502c.png)

Fixes #10474

#### Changes proposed in this Pull Request:

* Improve Calypso integration for relevant sites, and add admin ui autoupdate functionality.

#### Testing instructions:

* Install some plugins that are not in the repository (you should be able to select some from jurassic.ninja/specialops)
* Open your dev console and keep an eye out for js errors 
* Visit `/wp-admin/plugins.php?calypsoify=1`
* Test out the autoupdate plugin toggles
* Test out the bulk-select of auto-updates 
* Visit `/wp-admin/` to clear calypsoify skin.
